xgbgen/main.go

```diff
@@ -49,16 +46,26 @@ func main() {
 	c := newContext()
 	c.Morph(xmlBytes)
 
+	outFile, err := os.Create(fmt.Sprintf("%s/%s.go", *packageName, *packageName))
```

Comment:
Consider having a flag `output` rather than requiring the only output possible to be this filename.

Maybe someone wants to run the binary to standard out in order to test the output? Or to pipe it into some other tool.

Reply:
Fixed by 4760074eeb03154fee151b5bd99f64eae86d1426